dpkg/webodm/DEBIAN/postinst

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
+
+set -euxo
+
+export WEBODM_VERSION=1.9.7
+export WEBODM_USER=odm
+export WEBODM_USER_HOME="/home/${WEBODM_USER}"
+export WEBODM_DIR=/opt/WebODM
+export WEBODM_VENV_DIR="${WEBODM_DIR}/python3-venv"
+export NODEODM_DIR=/opt/nodeodm
+
+function abortRemove() {
+    echo "Unable to remove webodm."
+}
+
+function addNodeODMGPUIntelDockerGroups() {
+    DOCKER_COMPOSE_FILE="${WEBODM_DIR}/docker-compose.nodeodm.gpu.intel.yml"
+    for gid in $(id -G "${WEBODM_USER}"); do
+        SED_SEARCH="/^\s\{4\}group_add/{:start /${gid}$/!"
+        SED_SEARCH="${SED_SEARCH}{N;b start};/${gid}$/p}"
+        SED_OUT=$(sed -n "${SED_SEARCH}" "${DOCKER_COMPOSE_FILE}")
+        if [ -z "${SED_OUT}" ]; then
+            sed -i "/^\s\{4\}image/a \ \ \ \ group_add:\n\ \ \ \ \ \ - ${gid}" "${DOCKER_COMPOSE_FILE}"
+        fi
+    done
+}
+
+function buildNodeODMGPUIntelDockerImage() {
+    cd "${NODEODM_DIR}"
+
+    # We won't have to do this after the image is on Docker Hub.
```

Comment:
This is temporary, and can be removed after the [NodeODM PR](https://github.com/OpenDroneMap/NodeODM/pull/170) merges.